packages/addon-dev/sample-babel.config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "plugins": [
+    "@embroider/addon-dev/template-colocation-plugin",
+    ["@babel/plugin-proposal-decorators", { "legacy": true }],
+    "@babel/plugin-proposal-class-properties"
+  ]
+}
```

Comment:
why or why shouldn't we also ship compiled templates?

(for example: https://github.com/NullVoxPopuli/ember-addon-v2-typescript-demo/blob/main/my-addon/babel.config.cjs#L34-L39 )

```js
const { precompile } = require('@glimmer/compiler');

// ...

      resolve('babel-plugin-ember-template-compilation'),
      {
        precompile,
        enableLegacyModules: ['ember-cli-htmlbars'],
      },
    ],
```    

by not compiling away the templates, apps have more work to do, yea?

Reply:
Because the compiled format is deeply tied to the exact ember version that's going to consume it.

Reply:
gotchya, so this is probably only useful if you can guarantee that addons have the same ember version as apps :thinking: 

could be tricky

Reply:
It would be possible to make the wire format (the thing that you'd have if you use babel-plugin-ember-template-compilation) a stable public API, but somebody with knowhow would need to prioritize a little design work.